app/code/core/Mage/Core/Model/Resource/Abstract.php

```diff
@@ -44,6 +56,24 @@ public function __construct()
         $this->_construct();
     }
 
+    /**
+     * Default transaction isolation level defined in the database.
+     *
+     * @todo Default this value with whatever the database reports in SELECT @@GLOBAL.TX_ISOLATION
```

Comment:
Not sure how anyone else feels about this, but todos should be documented in an issue tracker, not in the code. Code based todos never get done.